test/c3/appframework/appframework_test.go

```diff
@@ -119,27 +105,29 @@ var _ = Describe("c3appfw test", func() {
 			// Verify MC is Ready and stays in ready state
 			testenv.VerifyMonitoringConsoleReady(deployment, deployment.GetName(), mc, testenvInstance)
 
+			// Upload V1 apps to S3 for IDXC
+			s3TestDirIdxc = "c3appfw-" + testenv.RandomDNSName(4)
```

Comment:
For IDXC and SHC, should we do as for MC and add a suffix in the name?
i.e "c3appfw-idxc" instead of "c3appfw-", like we have "c3appfw-mc-" on line 80